x/foundation/keeper/grpc_query.go

```diff
@@ -206,16 +206,16 @@ func (s queryServer) Grants(c context.Context, req *foundation.QueryGrantsReques
 		_, err := query.Paginate(grantStore, req.Pagination, func(key []byte, value []byte) error {
 			var authorization foundation.Authorization
 			if err := s.keeper.cdc.UnmarshalInterface(value, &authorization); err != nil {
-				return err
+				panic(err)
```

Comment:
Why did you change "return err" with "panic(err)"?
Is it because if you process it with return, you have to process each of them outside the function?

Reply:
The error is from `UnmarshalInterface`, which means the state is already corrupted. It's a critical bug and cause by the developer (in this case, me), so there is no reason to return it to the users. It also hides the cause of the error from the user, which is good from the security perspective.